file_formats/general.htm

```diff
@@ -127,7 +127,14 @@
               <td><code>.acm</code></td>
               <td>???</td>
               <td>&nbsp;</td>
-              <td>Music.</td>
+              <td>Music. ACM compressed sound files.</td>
+            </tr>
+            <tr>
+              <td>N/A</td>
+              <td><code>.acm</code></td>
```

Comment:
Really, not ogg?

Reply:
No, `.acm` file extension is shared by both ACM and Ogg encoded sound. Ogg encoding is used primarily by PST:EE music.